server/impl/api-client-impl.js

```diff
@@ -86,22 +87,30 @@ Collection.prototype.getCollectionCacheKeys = function (publisherId, depth) {
   return { storyCacheKeys, collectionCacheKeys };
 };
 
+Collection.prototype.isAutomated = function() {
+  return !!this.automated;
+};
+
+Collection.prototype.getChildCollections = function() {
+  return this.items && this.items.filter((i) => i.type === 'collection');
+};
+
+Collection.prototype.getChildStories = function() {
+  return this.items && this.items.filter((i) => i.type !== 'collection');
```

Comment:
Might be better to explicitly check for type story here?

Reply:
Sure 👍